go/tools/asthelpergen/asthelpergen.go

```diff
@@ -92,54 +92,41 @@ func newGenerator(mod *packages.Module, sizes types.Sizes, named *types.Named, g
 }
 
 func findImplementations(scope *types.Scope, iff *types.Interface, impl func(types.Type) error) error {
+	const OnlyReferences = false
+
 	for _, name := range scope.Names() {
 		obj := scope.Lookup(name)
 		if _, ok := obj.(*types.TypeName); !ok {
 			continue
 		}
 		baseType := obj.Type()
 		if types.Implements(baseType, iff) {
-			err := impl(baseType)
-			if err != nil {
+			if OnlyReferences {
```

Comment:
Doesn't this always evaluate to `false` considering `OnlyReferences` is a constant?

Reply:
I think the intent is to make it always `true` after we fix the current structs to only implement the `SQLNode` as pointers.
Currently switching it to `true` would panic the generator.

Reply:
Yes, we can't enable `onlyReferences` until we actually refactor our SQL AST to have only references! The functionality is here so we can check our implementation when it's eventually ready.